utils/src/treev2/quickPickWizard/GenericQuickPickStep.ts

```diff
@@ -0,0 +1,99 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as types from '../../../index';
+import * as vscode from 'vscode';
+import { getLastNode } from './QuickPickWizardContext';
+import { AzureWizardPromptStep } from '../../wizard/AzureWizardPromptStep';
+import { PickFilter } from './common/PickFilter';
+import { localize } from '../../localize';
+
+export interface GenericQuickPickOptions {
+    skipIfOne?: boolean;
+}
+
+export interface SkipIfOneQuickPickOptions extends GenericQuickPickOptions {

```

Comment:
In my mind that's basically the value--certain steps _always_ have `skipIfOne` as `true`, so I wanted to be maximally explicit--so that users of those steps understand they cannot set it to `false` (and, TS will complain if they try).